sass/directives/_visual-guides.scss

```diff
@@ -137,8 +130,9 @@ $visual-guide-breakpoint-small: 750px;
     width: $visual-guide-step-size;
     color: $text-color-dark;
 
-    span:after {
+    div:after {
```

Comment:
I just went ahead and pulled this and am now basing `:after` off `.visual-guide__step`. So a developer would only need to include `.visual-guide__step` in the template, and nothing else.